remix-tests/src/runTestSources.ts

```diff
@@ -20,15 +21,14 @@ const createWeb3Provider = async function () {
 /**
  * @dev Run tests from source of a test contract file (used for IDE)
  * @param contractSources Sources of contract
- * @param versionUrl url of selected compiler version to load
- * @param usingWorker if true, load compiler using web worker
+ * @param compilerConfig current compiler configuration
  * @param testCallback Test callback
  * @param resultCallback Result Callback
  * @param finalCallback Final Callback
  * @param importFileCb Import file callback
  * @param opts Options
  */
-export async function runTestSources(contractSources: SrcIfc, versionUrl: string, usingWorker: boolean, testCallback: Function, resultCallback: Function, finalCallback: any, importFileCb: Function, opts: Options) {
+export async function runTestSources(contractSources: SrcIfc, compilerConfig: CompilerConfiguration, testCallback: Function, resultCallback: Function, finalCallback: any, importFileCb: Function, opts: Options) {
```

Comment:
I think we would like to rethink this kind of API to use object instead. The more argument you have the harder it'll get to update. This kind of structure though is easier to maintain : 
```typescript
export async function runTestSources(options: Partial<RunTestSources> = defaultValue)
```
Like that if you want to add an argument it doesn't require any change in the current implementation

Reply:
Yes, I introduced single object for compiler configuration with that point of view. Maybe we can make the parameters more singular.